packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -538,4 +580,94 @@ function network.createMacvlanIface(baseIfname, linuxName, argsDev, argsIf)
 	return owrtInterfaceName, linuxName, owrtDeviceName
 end
 
+--! Create a static interface at runtime via ubus
+function network.createStatic(linuxBaseIfname)
+	local ipv4, ipv6 = network.primary_address()
+	local ubusIfaceName = network.sanitizeIfaceName(
+		network.LIME_UCI_IFNAME_PREFIX()..linuxBaseIfname.."_static")
+	local ifaceConf = {
+		name    = ubusIfaceName,
+		proto   = "static",
+		auto    = "1",
+		ifname  = linuxBaseIfname,
+		ipaddr  = ipv4:host():string(),
+		netmask = "255.255.255.255"
+	}
+
+	print('network', 'add_dynamic')
+	utils.dumptable(ifaceConf)
+
+	local libubus = require("ubus")
+	local ubus = libubus.connect()
+	ubus:call('network', 'add_dynamic', ifaceConf)
+	ubus:call('network.interface.'..ifaceConf.name, 'up', {})
+
+--! TODO: As of today ubus silently fails to properly setup the interface,
+--! subsequent status query return NO_DEVICE error
+--!  ubus -v call network.interface.lm_net_lm_net_wlan0_peer1_static status
+--!  {
+--!          "up": false,
+--!          "pending": false,
+--!          "available": false,
+--!          "autostart": true,
+--!          "dynamic": true,
+--!          "proto": "static",
+--!          "data": {
+--!
+--!          },
+--!          "errors": [
+--!                  {
+--!                          "subsystem": "interface",
+--!                          "code": "NO_DEVICE"
+--!                  }
+--!          ]
+--!  }
+--!
+--! ATM work around the problem configuring IP addresses via ip command
+
+	utils.unsafe_shell("ip link set up dev "..ifaceConf.ifname)
+	utils.unsafe_shell("ip address add "..ifaceConf.ipaddr.."/32 dev "..ifaceConf.ifname)
+
+	return ifaceConf.name
+end
+
+--! Create a vlan at runtime via ubus
+function network.createVlan(linuxBaseIfname, vid, vlanProtocol)
```

Comment:
Same comment as above: it would be convenient to rename this function so that the difference with `createVlanIface` is more clear.